src/Context/Internal/functions.php

```diff
@@ -6,79 +6,81 @@
 use Amp\Cancellation;
 use Amp\Future;
 use Amp\Parallel\Ipc;
+use Amp\Serialization\NativeSerializer;
 use Amp\Serialization\SerializationException;
-use Revolt\EventLoop;
+use Amp\Serialization\Serializer;
 
 /** @internal */
-function runContext(string $uri, string $key, Cancellation $connectCancellation, array $argv): void
-{
-    EventLoop::queue(function () use ($argv, $uri, $key, $connectCancellation): void {
-        /** @noinspection PhpUnusedLocalVariableInspection */
-        $argc = \count($argv);
+function runContext(
+    string $uri,
+    string $key,
+    Cancellation $connectCancellation,
+    array $argv,
+    Serializer $serializer = new NativeSerializer(),
+): void {
+    /** @noinspection PhpUnusedLocalVariableInspection */
+    $argc = \count($argv);
```

Comment:
Why did this code change in this PR?

Reply:
1) The call to `EventLoop::run()` was removed because the event loop is already running when called from `ForkContext`
2) Added the ability to pass a particular serializer instance, also used in `ForkContext`.